lntest/itest/lnd_test.go

```diff
@@ -14458,6 +14460,14 @@ func TestLightningNetworkDaemon(t *testing.T) {
 		ht.Fatalf("unable to request transaction notifications: %v", err)
 	}
 
+	// Connect chainbackend to miner.
+	if chainBackend.Name() != "neutrino" {
```

Comment:
could instead make the neutrino `ConnectMiner` a noop return

Reply:
yep that's cleaner, changed.

Reply:
Still not changed?

Reply:
Oops, my bad, now fixed😂